frameworks/cassandra/universe/config.json

```diff
@@ -205,7 +205,11 @@
                 }
 
             }
-        }
+        },
+        "custom_domain": {
+          "type": "string",
+          "description": "A custom domain to be used in place of autoip.dcos.thisdcos.directory. This can be used to expose the service securely outside of the cluster, but requires setting up external DNS. See the service documentation for details."
```

Comment:
Please reference a link where users can view more details for custom domain

Reply:
Hi @samvantran , can you please help where to give reference link. I tried in custom_domain description but it is not coming as clickable link.

Reply:
It doesn't have to be a clickable link. It can simply be a URL that a customer can paste in their browser. As for reference info, it would be great to also update the cassandra docs adding custom_domain info like here in zookeeper: https://docs.mesosphere.com/services/confluent-zookeeper/2.6.0-5.1.2e/security/#forwarding-dns-and-custom-domain